Sources/StructuredQueriesCore/QueryRepresentable/Codable+JSON.swift

```diff
@@ -44,3 +44,11 @@ extension JSONRepresentation: SQLiteType {
     String.typeAffinity
   }
 }
+
+private let jsonDecoder: JSONDecoder = {
```

Comment:
While we're in here should we also add a `jsonEncoder` that does a sorted keys in `DEBUG`?

Reply:
def!